### ISSUE TITLE
Shortcut keys for general options menu

### ISSUE THREAD

**Issue by i-need-to-tell-you-something** [laurent22/joplin#1564]
### Operating system

- Linux

### Application

- Desktop

---

General options menu cannot be exited by keyboard only. Escape key sounds reasonable. Alternatively they could be as follows:
```
Cancel            Alt+C
OK                Alt+O
Apply             Alt+A
```
These are something I as a KDE user am used to. Others might have different conventions. If they don't get their respective keys underlined then maybe add tooltips as well? Also why not Alt+C and Escape both for exiting.
